### ISSUE TITLE
[Question] Relocalization volume for an Anchor after session restart?

### ISSUE THREAD

**Issue by tangobravo** [google-ar/arcore-android-sdk#689]
I'm trying to wrap a consistent API around ARCore and ARKit. One area of difference is that the world coordinate system seems to have much lower consistency guarantees in ARCore vs ARKit.

For example relocalization is a separate state in ARKit that pops up after a session interruption. Once relocalized the world coordinate system should be aligned with where it was before the interruption. ARCore doesn't natively have a similar "global" relocalization state as far as I can tell, so world coordinates must be treated as unique to a particular `ArFrame`.

One workaround I thought of was to add an `ArAnchor` at the origin of the world coordinate system as soon as a fresh session starts tracking. We can then treat that as the origin of our "more ARKit compatible" world space, and report global relocalization state on the basis of the tracking state of that anchor (PAUSED = relocalizing).

I have two concerns with this approach:
1) #528 and #508 discuss how anchors placed early in a session lifecycle may be less accurate. I can accept a bit of drift during that early stage, as long as the drift of the anchor reduces as the world understanding improves. Is this the case or is that anchor somehow limited to the world understanding established at the point it was added?
2) What is the volume of relocalization for the anchor after a session is restarted? My tests are a bit inconclusive here. Should it switch to TRACKING as soon as there's any detected overlap between the point clouds from before and after a session interruption or does it require a camera pose close to the one where the anchor was initialized? Does plane detection state have any interaction with this? [NB: in this case the anchor is just placed on the session, with `ArSession_acquireNewAnchor`]

So basically the question boils down to "is adding a single anchor as a world origin a reasonable thing to do?". We will expose a way of adding additional anchors and recommend them as the best way for placing content in general, but for some applications where relative anchor movement is a problem it would be nicer to work in a single world space.
